### ISSUE TITLE
Add to apt (debian based linux package manager)

### ISSUE THREAD

**Issue by mrmuminov** [rgwood/systemctl-tui#24]
Hi, can you add to apt-get package manager?

**Comment by rgwood** [comment]
No, sorry. I don’t have any interest in dealing with distro-specific packaging (but other people can try if they want).

**Comment by Tampa** [comment]
Yeah let's just ignore the most common distro out there and instead require figuring out how to install rust and require compiling, because spending the ten minutes to copy the deb packaging template for github actions is too much work. C'mon do you want adoption of this or not? None of the current installation instructions are complete enough to get anyone from 0 to 100, because rustup doesn't come pre-installed on any debian-like distros. I'll work it out and PR the readme, but honestly telling people to just do whatever don't expect anyone to bother.

**Comment by rgwood** [comment]
> spending the ten minutes to copy the deb packaging template for github actions is too much work

I think on some level you know it's more work than that, because instead of doing that in your PR you just told people to install Rust and compile from source.

**Comment by rgwood** [comment]
I probably shouldn't have closed this issue earlier, my bad. I'm open to improvements for the Ubuntu+Debian situation (just don't want to make them myself).

**Comment by antonsizikov** [comment]
I too was a bit confused when I wanted to install tour app in Ubuntu.
I adapted the install/update script from to [lazydocker](https://github.com/jesseduffield/lazydocker). It downloads the latest version from github for the desired architecture, unzips, installs the application, and then deletes the archive. Should work on all distributions. 
@rgwood If I open a PR adding it to the repository and add note to the Readme, will you approve it?

**Comment by rgwood** [comment]
@antonsizikov Yes, I would probably accept a PR like that. Thanks!

**Comment by antonsizikov** [comment]
So here is the PR https://github.com/rgwood/systemctl-tui/pull/37
